custom/icds_reports/reports/governance_apis.py

```diff
@@ -143,6 +154,15 @@ def get_cbe_data(length, year, month, order, query_filters):
         'num_other_beneficiaries_2',
     )
 
-    # To apply pagination on database query with offset and limit
+    def get_value_or_data_not_entered(value):
```

Comment:
This looks a lot like functions we already have in the utils file. Any reason not to use those?

Reply:
In fact it looks like you have already defined a similar one in this file. At worst, if this function does not exist it would be good to move it to the utils file and use the same one in every place instead of redefining it multiple times here.

Reply:
removed this util function.